Source/ACE/Network/GameEvent/Events/GameEventDeathNotice.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ACE.Network.GameEvent.Events
```

Comment:
This is also needed for player death notices, so this code would be useful to have as a separate PR, for Merging.